vendors/nxp/boards/lpc54018iotmodule/aws_demos/config_files/FreeRTOSConfig.h

```diff
@@ -58,7 +58,7 @@
 #define configTICK_RATE_HZ                           ( ( TickType_t ) 1000 )
 #define configMAX_PRIORITIES                         ( 7 )
 #define configMINIMAL_STACK_SIZE                     ( ( uint16_t ) 90 )
-#define configTOTAL_HEAP_SIZE                        ( ( size_t ) ( 100 * 1024 ) )
+#define configTOTAL_HEAP_SIZE                        ( ( size_t ) ( 150 * 1024 ) )
```

Comment:
You also want to the heap size for tests in this PR, right?

Reply:
We don't run the OTA demo in tests project, but I'll make them the same to keep it consistent.